front-packages/akeneo-design-system/src/components/Input/MultiSelectInput/MultiSelectInput.tsx

```diff
@@ -245,7 +245,7 @@ const MultiSelectInput = ({
           ref={inputRef}
           id={id}
           placeholder={placeholder}
-          value={value.map(chipCode => indexedChips[chipCode])}
+          value={value.map(chipCode => indexedChips[chipCode] ?? {code: chipCode, label:chipCode})}
```

Comment:
```suggestion
          value={value.map(chipCode => indexedChips[chipCode] ?? {code: chipCode, label: chipCode})}
```

Maybe add a test about it the added test do not test it ?

Reply:
fixed